### PR TITLE
cleans up collections and extracts marketplace component

### PR DESCRIPTION
**Pull request by cosmoburn**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# Mandatory info for starting the app
```
Comment (andyault) [inline .env.example]: was this deleted on purpose?

File: .gitignore [hunk 2]
```diff
@@ -14,6 +14,7 @@
 # misc
 .DS_Store
 .env
+.env.example
```
Comment (andyault) [inline .gitignore]: was this added on purpose? I don't think we wanna ignore the .env.example

File: src/Marketplace/Marketplace.jsx [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+  logo: {
+    height: "115px",
+    width: "115px",
+    borderRadius: "50%",
+    // positon: "relative",
+    // marginTop: "-80px",
+    border: "solid 4px white",
+  },
+  text: {
+    color: "#041836",
```
Comment (andyault) [inline src/Marketplace/Marketplace.jsx]: would like to see some of these settings (especially colors) moved to some kind of constants file eventually

File: src/helpers/collections.js [hunk 4]
```diff
@@ -1,70 +1,14 @@
...
-        "https://lh3.googleusercontent.com/LIpf9z6Ux8uxn69auBME9FCTXpXqSYFo8ZLO1GaM8T7S3hiKScHaClXe0ZdhTv5br6FE2g5i-J5SobhKFsYfe6CIMCv-UfnrlYFWOM4=s0",
-      name: "CyberKongz",
-      addrs: "0x57a204AA1042f6E66DD7730813f4024114d74f37",
-    },
-    {
-      image:
-        "https://lh3.googleusercontent.com/LIov33kogXOK4XZd2ESj29sqm_Hww5JSdO7AFn5wjt8xgnJJ0UpNV9yITqxra3s_LMEW1AnnrgOVB_hDpjJRA1uF4skI5Sdi_9rULi8=s0",
-      name: "Cool Cats NFT",
-      addrs: "0x1A92f7381B9F03921564a437210bB9396471050C",
+      name: "Punky Ape Yacht Club",
```
Comment (andyault) [inline src/helpers/collections.js]: typo

File: src/Marketplace/Marketplace.jsx (hunk 3 above)
Comment (cosmoburn) [reply src/Marketplace/Marketplace.jsx]: Absolutely will, just trying to get some stuff out for now so the other guys can work on it. I added emotion as well on my next fork

**Pull request closed without merging**